site/docs/v1/tech/core-concepts/_fusionauth_roles.adoc

```diff
@@ -0,0 +1,59 @@
+.FusionAuth application roles
+[cols="1,2,1"]
+|===
+| Name | Id | Description
+
+| `admin` | `631ecd9d8d404c13827780cedb8236e2` | Can manage everything, including creating new users with administrator privileges.
+| `api_key_manager` | `631ecd9d8d404c13827780cedb8236e3` | Can add and edit API keys.
+| `application_deleter` | `631ecd9d8d404c13827780cedb8236e4` | Can delete applications.
+| `application_manager` | `631ecd9d8d404c13827780cedb8236e5` | Can add and edit applications.
+| `audit_log_viewer` | `631ecd9d8d404c13827780cedb8236e6` | Can view audit logs.
+| `connector_deleter` | `631ecd9d8d404c13827780cedb823700` | Can delete Connectors.
+| `connector_manager` | `631ecd9d8d404c13827780cedb823701` | Can add and edit Connectors.
+| `consent_deleter` | `631ecd9d8d404c13827780cedb8236fc` | Can delete consents.
+| `consent_manager` | `631ecd9d8d404c13827780cedb8236fd` | Can addd and edit consents.
+| `email_template_manager` | `631ecd9d8d404c13827780cedb8236e7` | Can add and edit email templates.
+| `event_log_viewer` | `631ecd9d8d404c13827780cedb8236fa` | Can view the event log.
+| `form_deleter` | `631ecd9d8d404c13827780cedb823702` | Can delete forms and form fields.
+| `form_manager` | `631ecd9d8d404c13827780cedb823703` | Can add and edit forms and form fields.
+| `group_deleter` | `631ecd9d8d404c13827780cedb8236f6` | Can delete groups.
+| `group_manager` | `631ecd9d8d404c13827780cedb8236f5` | Can add and edit groups.
+| `key_manager` | `631ecd9d8d404c13827780cedb8236fb` | Can add and edit keys.
+| `lambda_manager` | `631ecd9d8d404c13827780cedb8236f9` | Can add and edit lambdas.
+| `reactor_manager` | `631ecd9d8d404c13827780cedb8236ff` | Can add and edit reactor settings.
+| `report_viewer` | `631ecd9d8d404c13827780cedb8236e8` | Can view reports.
+| `system_manager` | `631ecd9d8d404c13827780cedb8236e9` | Can add and edit system configuration.
+| `tenant_deleter` | `631ecd9d8d404c13827780cedb8236f8` | Can delete tenants.
+| `tenant_manager` | `631ecd9d8d404c13827780cedb8236f7` | Can add and edit tenants.
+| `theme_manager` | `631ecd9d8d404c13827780cedb8236fe` | Can add and edit themes.
+| `user_action_deleter` | `631ecd9d8d404c13827780cedb8236f0` | Can delete user actions.
+| `user_action_manager` | `631ecd9d8d404c13827780cedb8236f1` | Can add and edit user actions.
+| `user_deleter` | `631ecd9d8d404c13827780cedb8236f2` | Can delete users.
+| `user_manager` | `631ecd9d8d404c13827780cedb8236f3` | Can add and edit users. 
+| `user_support_manager` | `631ecd9d8d404c13827780cedb823704` | Allows for a limited scope of user management. See below.
+| `user_support_viewer` | `631ecd9d8d404c13827780cedb823705` | Can view user information.
```

Comment:
We don't add roles real often, so not sure how far back we need to go - but these two will be added in 1.23.0. 

So perhaps add `Since 1.23.0`. 

The other "semi-recent" ones would be `reactor_manager`, `consent_*` and `connector_*` - since this is only for reference, may not worth mentioning version at all - or at least not too far back? Thoughts? 

If the role doesn't exist - it won't exist.

Reply:
I like going back at least 1-2 versions. I'll look in the code and see if I can figure out the older ones.

Reply:
Went back to 1.15 (reactor manager).